src/crab/ebpf_checker.cpp

```
@@ -52,14 +52,46 @@ class EbpfChecker final {
+    //
+    //  - "dynamic" regions have a ceiling that *does* depend on the
+    //    allocation site: T_SHARED's shared_region_size and T_ALLOC_MEM's
+    //    alloc_mem_size are kind variables on the RegPack, set when the
+    //    pointer was produced (map lookup, ringbuf_reserve, ...). The
+    //    TypeEncoding adds nothing the size variable does not already
+    //    carry, so the dynamic-region check takes the size directly.
+
+    // Static-region bounds, ceiling chosen by `type`:
+    //   T_STACK / T_CTX -> their AnalysisContext-derived ceilings;
+    //   T_PACKET        -> the loose max_packet_size constant. Use this
+    //                      T_PACKET form *only* for pointer-comparison
```

Cppcheck (2.20.0)

[style] The function 'entry_label' is never used.

(unusedFunction)